mx.c

```diff
@@ -578,7 +578,7 @@ int mx_mbox_close(struct Context **ptr)
   if (m->readonly || m->dontwrite || m->append)
   {
     mx_fastclose_mailbox(m);
-    FREE(ptr);
+    ctx_free(ptr);
```

Comment:
The `Context` must be freed using its `ctx_free()` function.
That's my fault :-)